protocol/daemons/liquidation/client/client.go

```diff
@@ -236,6 +264,13 @@ func SendLiquidatableSubaccountIds(
 		metrics.Latency,
 	)
 
+	telemetry.ModuleSetGauge(
```

Comment:
Should we stat the number of non-liquidatable subaccounts somewhere?

Reply:
we can get that with `allSubaccounts - liquidatableSubaccounts`